health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/fetchFromMicroplan.js

```
@@ -0,0 +1,166 @@
+  ];
+
+  useEffect(() => {
+    const fetchTemplates = async () => {
+      if (data && data?.updatedCampaignData) {
+        if (currentStep === 0) {
+          setCurrentStep((prev) => prev + 1);
+
+        }
+  
+        try {
+          const { hierarchyType, id, tenantId } = data?.updatedCampaignData;
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)